cmd/upgrade.go

```diff
@@ -0,0 +1,201 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"strconv"
+	"strings"
+
+	_ "github.com/lib/pq" // pg driver
+	"github.com/spf13/cobra"
+)
+
+// upgradeCmd represents the validate command
+var upgradeCmd = &cobra.Command{
+	Use:   "upgrade",
+	Short: "Upgrade all mirror registry images.",
+	Run: func(cmd *cobra.Command, args []string) {
+		upgrade()
+	},
+}
+
+func init() {
+
+	// Add upgrade command
+	rootCmd.AddCommand(upgradeCmd)
+
+	upgradeCmd.Flags().StringVarP(&targetHostname, "targetHostname", "H", getFQDN(), "The hostname of the target you wish to install Quay to. This defaults to $HOST")
+	upgradeCmd.Flags().StringVarP(&targetUsername, "targetUsername", "u", os.Getenv("USER"), "The user on the target host which will be used for SSH. This defaults to $USER")
+	upgradeCmd.Flags().StringVarP(&sshKey, "ssh-key", "k", os.Getenv("HOME")+"/.ssh/quay_installer", "The path of your ssh identity key. This defaults to ~/.ssh/quay_installer")
+
+	upgradeCmd.Flags().StringVarP(&quayHostname, "quayHostname", "", "", "The value to set SERVER_HOSTNAME in the Quay config.yaml. This defaults to <targetHostname>:8443")
```

Comment:
Do we need this in the upgade?

Reply:
I think so...if we remove it, there wouldn't be any way for users who remotely installed mirror-registry to upgrade their remote installations. Need the SSH info.